.rubocop.yml

```diff
@@ -11,9 +11,13 @@ Style/Documentation:
 
 Metrics/MethodLength:
   Max: 15
+  Exclude:
```

Comment:
Well. it excludes it from just one metric. But you are right, this should not be necessary since this is new code. @cduongt Do we really have a good enough reason to exclude the check rather than satisfy it?

Reply:
`MethodLength` and `AbcSize` are one of those main metrices that keeps code clean, divided into multiple smaller methods and that way testable. And as you pointed out it's a new code.

Reply:
:+1: